### PR TITLE
blocking is bad

### PR DESCRIPTION
**Pull request by galganif**

### CODE REVIEW

File: java/arcs/android/storage/service/StorageServiceManager.kt [hunk 1]
```diff
@@ -36,7 +37,7 @@ class StorageServiceManager(
     private val coroutineContext = parentCoroutineContext + CoroutineName("StorageServiceManager")
 
     override fun clearAll(resultCallback: IResultCallback) {
-        runBlocking(coroutineContext) {
+        CoroutineScope(coroutineContext).launch {
```
Comment (jblebrun) [inline java/arcs/android/storage/service/StorageServiceManager.kt:40]: should we just create one `val scope = CoroutineScope(coroutineContext)` and just use `scope.launch`? (Not sure how much it matters, but it does give us an owned scope to eventually cancel somehow if needed)

Comment (jasonwyatt) [reply java/arcs/android/storage/service/StorageServiceManager.kt:40]: yeah, that's a good plan.

**Comment by jblebrun** [comment]
The AIDL calls should probably be oneway as well right? But I don't think we need that now, if others agree, I'll file a bug

**Comment by galganif** [comment]
> The AIDL calls should probably be oneway as well right? But I don't think we need that now, if others agree, I'll file a bug

ack

**Pull request merged**